R/opendp/src/convert.c

```diff
@@ -93,11 +93,15 @@ void *sexp_to_voidptr(SEXP input, SEXP rust_type)
     PROTECT(input);
     PROTECT(rust_type);
 
-    if (str_equal(sexp_to_charptr(get_origin(rust_type)), "Option")) {
-        if (input == R_NilValue) {
+    if (str_equal(sexp_to_charptr(get_origin(rust_type)), "Option"))
+    {
+        if (input == R_NilValue)
+        {
             UNPROTECT(2);
             return NULL;
-        } else {
+        }
+        else
+        {
```

Comment:
Formatting changes don't seem to be related to PR... but not causing a problem either.